src/terrain/routes/schemas/instantlaunches.clj

```diff
@@ -3,11 +3,36 @@
         [schema.core :only [defschema Any Keyword optional-key]])
   (:import [java.util UUID]))
 
+(def InstantLaunchIDParam (describe String "The ID of the instant launch"))
+
+(def ListInstantLaunchesSummary "Lists instant launches")
+(def ListInstantLaunchesDescription "Lists the instant launches available in the system")
+(def AddInstantLaunchSummary "Adds an instant launch")
+(def AddInstantLaunchDescription "Creates a new instant launch")
+(def GetInstantLaunchSummary "Returns an instant launch")
+(def GetInstantLaunchDescription "Returns an instant launch based on its UUID")
+(def UpdateInstantLaunchSummary "Updates an instant launch")
+(def UpdateInstantLaunchDescription "Updates an instant launch's fields")
+(def DeleteInstantLaunchSummary "Deletes an instant launch")
+(def DeleteInstantLaunchDescription "Deletes an instant launch by its UUID")
+(def AddLatestILMappingsDefaultsSummary "Adds a new mapping as the latest")
+(def AddLatestILMappingsDefaultsDescription "Adds a new mapping as the latest, incrementing the version number")
+(def UpdateLatestILMappingsDefaultsSummary "Updates the latest mapping")
+(def UpdateLatestILMappingsDefaultsDescription "Updates the latest mapping without changing the version number")
+(def DeleteLatestILMappingsDefaultsSummary "Deletes the latest mapping")
+(def DeleteLatestILMappingsDefaultsDescription "Deletes the latest mapping, effectively rolling back to the previous version")
+(def LatestILMappingsDefaultsSummary "The latest defaults for instant launch mappings")
+(def LatestILMappingsDefaultsDescription "The latest defaults for instant launch mappings,
+   which determine which files can be used with a particular instant launch")
+
 (defschema InstantLaunch
-  {:id              (describe UUID "The UUID of the instant launch")
-   :quick_launch_id (describe UUID "The UUID for the quick launch used for the instant launch")
-   :added_by        (describe String "The username of the user that created the instant launch")
-   :added_on        (describe String "The date and time when the instant launch was created")})
+  {(optional-key :id)              (describe UUID "The UUID of the instant launch")
+   :quick_launch_id                (describe UUID "The UUID for the quick launch used for the instant launch")
+   (optional-key :added_by)        (describe String "The username of the user that created the instant launch")
```

Comment:
This is minor, but `user who created the instant launch` is probably more correct.

Reply:
Oops, yup. Should be fixed now.